noxfile.py

```diff
@@ -375,6 +375,15 @@ def cover(session):
     session.install("coverage", "pytest-cov")
     session.run("coverage", "report", "--show-missing", "--fail-under=96")
 
+    # Make sure there is no dead code in our test directories.
+    session.run(
+        "coverage",
+        "report",
+        "--show-missing",
+        "--include=tests/unit/*",
+        "--fail-under=100",
+    )
+
```

Comment:
Does this coverage addition make the previous coverage run obsolete?

Reply:
No, this one is only for our test code. We still aren't quite at 100% in the actual library code. This makes sure we actually run all of the tests. It's easy to forget a `test_` prefix or have some try/except block that never actually gets run.